### PR TITLE
Update certs on image build

### LINKED ISSUE #4722: Include `apt install ca-certificates` during docker build process
By not updating the certificate bundles, we are relying on the upstream golang container to have a build process that is providing us with the most recent certs.  Their is a potential for our certs in our containers to not be up to date causing errors going forwards.

### PR DESCRIPTION
**Pull request by ysimonson**
Closes #4722

### CODE REVIEW

File: etc/reset.py [hunk 1]
```diff
@@ -19,6 +19,8 @@
     "secrets",
     "clusterrole",
     "clusterrolebinding",
+    "roles.rbac.authorization.k8s.io",
```
Comment (seslattery) [inline etc/reset.py]: I don't have context on why this change is necessary, but at first glance it seems fine.

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # syntax=docker/dockerfile:1.0-experimental
 ARG GO_VERSION
 FROM golang:${GO_VERSION}
+RUN apt update && apt install ca-certificates
```
Comment (seslattery) [inline Dockerfile]: Might be overkill for single package, but `&& rm -rf /var/lib/apt/lists/*` is best practice.

Comment (ysimonson) [reply Dockerfile]: I don't think that makes sense here, because this is a throwaway build container, i.e. the image isn't pushed, and only selective assets from the image are copied into other images.

File: etc/reset.py (hunk 1 above)
Comment (ysimonson) [reply etc/reset.py]: You can use `./etc/reset.py` to fully reset a development deployment of pachyderm. It looks like there were some recent changes on docker for mac that now make it necessary to delete roles and rolebindings, and it was blocking testing of these changes.
